packages/@lwc/synthetic-shadow/src/env/slot.ts

```diff
@@ -1,25 +1,9 @@
 /*
- * Copyright (c) 2018, salesforce.com, inc.
+ * Copyright (c) 2023, Salesforce.com, inc.
  * All rights reserved.
  * SPDX-License-Identifier: MIT
  * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
  */
 
-let assignedNodes: (options?: AssignedNodesOptions) => Node[],
-    assignedElements: (options?: AssignedNodesOptions) => Element[];
-if (typeof HTMLSlotElement !== 'undefined') {
-    assignedNodes = HTMLSlotElement.prototype.assignedNodes;
-    assignedElements = HTMLSlotElement.prototype.assignedElements;
-} else {
-    assignedNodes = () => {
-        throw new TypeError(
-            "assignedNodes() is not supported in current browser. Load the @lwc/synthetic-shadow polyfill to start using <slot> elements in your Lightning Web Component's template"
-        );
-    };
-    assignedElements = () => {
-        throw new TypeError(
-            "assignedElements() is not supported in current browser. Load the @lwc/synthetic-shadow polyfill to start using <slot> elements in your Lightning Web Component's template"
-        );
-    };
-}
-export { assignedNodes, assignedElements };
+export const assignedNodes = HTMLSlotElement.prototype.assignedNodes;
+export const assignedElements = HTMLSlotElement.prototype.assignedElements;
```

Comment:
I wonder how many of these types of assignments could be refactored away in the `synthetic-shadow` package now that more and more of them are just extra layers of assignments with no added wrapper functionality. (e.g. `window.ts`).

Reply:
I'm sure a lot of it could, yeah. I held back from heavy refactoring, though, since synthetic shadow is deprecated, and minifiers will clean up a lot of these abstraction layers anyway.